docs/2.0. APIs.md

```diff
@@ -37,9 +37,10 @@ https://<ip address or hostname>:<port>/x-nmos/auth/<api version>/
 The Authorization Server MUST implement an endpoint at the following path. This path immediately follows the port and
 MUST NOT be prefixed by any additional path, including but not limited to 'x-nmos'.
 
-*   **/.well-known/oauth-authorization-server** - This path MUST return a response matching the requirements of OAuth
-2.0 Authorization Server Metadata [RFC 8414][RFC-8414]. This MUST include the optional `jwks_uri`,
-`registration_endpoint` and `revocation_endpoint` parameters.
+*   **/.well-known/oauth-authorization-server[/issuer]** - This path MUST return a response matching the requirements
```

Comment:
Could you note explicitly that this one path does not (cannot) follow the usual requirement for `x-nmos` etc in the path? I realise this is stated more explicitly in another section.